app/models/activity_log.rb

```diff
@@ -0,0 +1,10 @@
+class ActivityLog < ApplicationRecord
+  def self.filter(params = {})
+    scope = all
+    scope = scope.where(http_format: params[:http_format]) if params[:http_format].present?
+    scope = scope.where(http_method: params[:http_method]) if params[:http_method].present?
+    scope = scope.where("DATE(created_at) >= ?", params[:start_date]) if params[:start_date].present?
+    scope = scope.where("DATE(created_at) <= ?", params[:end_date])   if params[:end_date].present?
```

Comment:
@radinreth please be careful with datetime following the application timezone vs the database timezone, in e.g: 2021-09-01 00:00:00 in timezone(ICT +7) but store in database as 2021-08-31 17:00:00 in database timezone (UTC +0) -> and when we cast DATE(created_at) of this date you will see what different of the result

Reply:
scorecard currently use application's timezone (Bankok)

![image](https://user-images.githubusercontent.com/5484758/132481856-b0d2a503-540a-4c93-b203-896f0e53cbe2.png)